Formula/homeassistant-cli.rb

```diff
@@ -175,8 +175,7 @@ class HomeassistantCli < Formula
   def install
     virtualenv_install_with_resources
     bin.install_symlink libexec/"bin/hass-cli"
-    (bash_completion/"hass-cli").write Utils.safe_popen_read(bin/"hass-cli", "completion", "bash")
-    (zsh_completion/"_hass-cli").write Utils.safe_popen_read(bin/"hass-cli", "completion", "zsh")
+    generate_completions(base_name: "hass-cli", shells: [:bash, :zsh], binary: bin/"hass-cli")
```

Comment:
Does this work? I think since `binary` defaults to `bin/base_name` this should be fine. Also, see https://github.com/Homebrew/brew/pull/13536#pullrequestreview-1037685543

```suggestion
    generate_completions(base_name: "hass-cli", shells: [:bash, :zsh])
```